src/validation.h

```diff
@@ -208,19 +208,26 @@ struct PackageMempoolAcceptResult
 };
 
 /**
- * Try to add a transaction to the mempool. This is an internal function and is
- * exposed only for testing. Client code should use ChainstateManager::ProcessTransaction()
+ * Try to add a transaction to the mempool. This is an internal function and is exposed only for testing.
+ * Client code should use ChainstateManager::ProcessTransaction()
  *
- * @param[in]  active_chainstate  Reference to the active chainstate.
+ * @param[in]  chainparams        Chain paramenters.
  * @param[in]  pool               Reference to the node's mempool.
+ * @param[in]  active_chainstate  Reference to the active chainstate.
  * @param[in]  tx                 The transaction to submit for mempool acceptance.
+ * @param[in]  accept_time        The timestamp for adding the transaction to the mempool. Usually
+ *                                the current system time, but may be different (eg when loading
+ *                                mempool txs from disk on startup).
```

Comment:
It's generally not a good idea to document how the function is called in the function comment (since that may change over time, and this comment would be out of date:

```suggestion
 *                                the current system time, but may be different.
```

Reply:
Makes sense. Comment removed.